src/test/java/core/basesyntax/TruckTest.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+class TruckTest extends MachineTest {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working.");
+    }
+}
```

Comment:
each file should end with exactly one empty line, which will not be shown on github
if there's no empty line at the end, red symbol like this will appear

Reply:
each file should end with exactly one empty line, which will not be shown on github
if there's no empty line at the end, red symbol like this will appear. How can I fix this and still pass the test? Is it possible to change the settings?

Reply:
There is one empty line in all files